app/services/streaming/streaming-view.ts

```diff
@@ -45,16 +37,16 @@ export class StreamInfoView extends ViewHandler<IStreamingServiceState> {
     return this.getServiceViews(TwitterService);
   }
 
-  private get incrementalRolloutView() {
-    return this.getServiceViews(IncrementalRolloutService);
+  private get streamingState() {
+    return Services.StreamingService.state;
```

Comment:
does the service provider work like this in views and other services?

Reply:
Yes, it does. But I probably should rollback it to the previous style until we come with some convention about how we should inject services

Reply:
agreed, especially since views are explicitly only supposed to be able to access other views and state